packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -240,6 +241,10 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     if (isOpen && !prevState.isOpen && onMenuOpen) {
       onMenuOpen();
     }
+
+    if (!isOpen && prevState.isOpen && onMenuDismissed) {
+      onMenuDismissed();
```

Comment:
I'll defer to @jspurlin, but shouldn't we put this the _onDismiss method?

Reply:
It doesn't get called every time we dismiss a menu, for example, when pressing tab

Reply:
that's the reason I put it here similar to onMenuOpen

Reply:
Nit: can we make this onMenuDismiss to make it consistent with onMenuOpen?

Reply:
The name suggested here aligns with the onMenuDismissed for the ContextualMenu.

It does seems weird that this gets called on the next Render and not when we know we are dismissing the menu (e.g. setting the opened state to false). Consider creating a function to set the open state and call onMenuDismissed when you are setting open to false

Reply:
But the same thing could be said about the onMenuOpen function so this isn't a must, just a consideration... You could actually pull onMenuOpen into the same function if you did that work

Reply:
Talked with @christiango (who added onMenuOpen) offline about this let's keep this code as is for now